src/sage/groups/generic.py

```diff
@@ -1257,14 +1264,23 @@ def order_from_multiple(P, m, plist=None, factorization=None, check=True,
     elif operation in addition_names:
         identity = P.parent()(0)
     else:
-        raise ValueError("unknown group operation")
+        if identity is None or inverse is None or op is None:
+            raise ValueError("identity, inverse and operation must all be specified")
+
+    def _multiple(A, B):
+        return multiple(A,
+                        B,
+                        operation=operation,
+                        identity=identity,
+                        inverse=inverse,
+                        op=op)
 
     if P == identity:
         return Z.one()
 
     M = Z(m)
     if check:
-        assert multiple(P, M, operation=operation) == identity
+        assert _multiple(P, M) == identity
```

Comment:
Hm, this should probably be a `ValueError` since it can be triggered by invalid user input, right?